packages/react-components/theme-designer/src/ThemeDesigner.tsx

```diff
@@ -41,74 +51,84 @@ const useStyles = makeStyles({
 export const ThemeDesigner: React.FC<ThemeDesignerProps> = props => {
   const styles = useStyles();
 
-  const [theme, setTheme] = React.useState<string>('Custom');
-  const [keyColor, setKeyColor] = React.useState<string>('#006bc7');
-  const [hueTorsion, setHueTorsion] = React.useState<number>(0);
-  const [darkCp, setDarkCp] = React.useState<number>(2 / 3);
-  const [lightCp, setLightCp] = React.useState<number>(1 / 3);
-
-  const debouncedKeyColor = useDebounce(keyColor, 400);
-
-  const customTheme = React.useMemo(() => {
+  const createCustomTheme = ({
+    keyColor,
+    hueTorsion,
+    lightCp,
+    darkCp,
+    isDark,
```

Comment:
_nit:_ Ditto about alphabetical order

```suggestion
    darkCp,
    hueTorsion,
    isDark,
    keyColor,
    lightCp,
```